assets/js/netlify/makeHTMLFromBodyContent.js

```diff
@@ -8,6 +8,8 @@ function makeHTMLFromBodyContent(bodyContent, variables, imageData) {
   let content = bodyContent?.replaceAll(/<hr>/g, "***")
       .replaceAll("{{'", '')
       .replaceAll("' | relative_url}}", '')
+      .replaceAll("'| relative_url}}", '')
+      .replaceAll("'|relative_url}}", '')
```

Comment:
You could use a regex for this, too - that would also account for the ability to have spaces before the `}}`, and to have an arbitrary number of spaces:

https://regex101.com/r/EjI2lY/2

```suggestion
      .replaceAll(/\s*\|\s*relative_url\s*}}/, '')
```